src/api/app/views/webui2/webui/user/_register_user.html.haml

```diff
@@ -0,0 +1,11 @@
+.modal.fade#signup-modal{ tabindex: -1, role: 'dialog', aria: { labelledby: 'signup-modal-label', hidden: true } }
```

Comment:
I believe we shouldn't use a modal for this form. It's quite "heavy" for a modal. I know that we did use modals a lot in the pages we previously migrated, but I believe we took the wrong decision there. This should be on its own page. It has advantages such as better user feedback when something goes wrong. We don't have anything for that in our modals.

Reply:
@dmarcoux okay, i will revert it to a detached view